seaborn/categorical.py

```diff
@@ -2423,26 +2461,30 @@ def boxenplot(
     showfliers : bool, optional
         If False, suppress the plotting of outliers.
     {ax_in}
+    box_kws: key, value mappings
+        Other keyword arguments are passed through to
```

Comment:
This isn't "other keyword arguments", please rephrase as `"Keyword arguments for the box artists; passed to ..."`